libs/portal/shared/features/form/ui/text-area-field.tsx

```
@@ -0,0 +1,89 @@
+                <p>
+                  {label}
+                  <span className="text-red-600 ml-1">
+                    {isRequired ? '*' : null}
+                  </span>
+                </p>
+              </Typography>
+            </FormLabel>
+            <FormControl>
+              <div className="w-full self-stretch rounded-md flex items-center gap-3">
+                <div className="py-1.5 grow basis-0 flex items-start gap-3 w-full">
+                  <img src={isDisabled ? tildeIcon : imgSrc} className={cn('size-6', isDisabled && 'opacity-25')} />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)